VnE/Source/MainWindow.py

```
@@ -235,9 +245,40 @@ def select(self, pos):
+        glBindFramebuffer(GL_FRAMEBUFFER, self.screen_fbo)
+        glFramebufferRenderbuffer(GL_FRAMEBUFFER, GL_DEPTH_STENCIL_ATTACHMENT, GL_RENDERBUFFER, self.screen_dsrbo)
+        glFramebufferRenderbuffer(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, GL_RENDERBUFFER, self.screen_crbo)
+        glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT)
+        glDrawBuffers(1, GL_COLOR_ATTACHMENT0)
+        self.facade.drawScene(self.scene)
+        glFlush()
+        glBindFramebuffer(GL_FRAMEBUFFER, self.screen_fbo)
+        glReadBuffer(GL_COLOR_ATTACHMENT0)
+
+        c = np.zeros((res[1], res[0], 1), dtype=np.uint32)
+        glReadPixels(0, 0, *res, GL_RGBA, GL_UNSIGNED_INT_8_8_8_8_REV, c)
```

Ruff (0.15.6)

[error] `glReadPixels` may be undefined, or defined from star imports

(F405)